src/components/profile-view/profile-view.jsx

```diff
@@ -0,0 +1,150 @@
+import  {useState}  from "react";
+import { MovieCard } from "../movie-card/movie-card"
+import { Container, Row, Col, Form, Button } from "react-bootstrap";
+
+export const ProfileView = ({ user, token, movies, setUser }) => {
+
+    const [username, setUsername] = useState(user.name)
+    const [password, setPassword] = useState(user.password)
+    const [email, setEmail] = useState(user.email)
+    const [birthday, setBirthday] = useState(user.birthday)
```

Comment:
Those state variables that are attributed to inputs should be initially empty. Giving them the current values from the user object doesn't help with submitting the edit form.

Reply:
I did the above two point and the app is working now.  Thank you!